build/configs/artik053/tools/openocd/artik053.cfg

```diff
@@ -1,6 +1,13 @@
+source [find romfs_flashing.cfg]
 source [find artik053-usb-ftdi.cfg]
 source [find s5jt200.cfg]
-source [find partition_map.cfg]
+global romfs_partition_enable
+if { $romfs_partition_enable == 1 } {
+	source [find partition_map_romfs.cfg]
+}
+if { $romfs_partition_enable == 0 } {
+	source [find partition_map.cfg]
+}
```

Comment:
Instead of keeping 2 different files, can you manage in a single cfg file. Check if you can include/exclude romfs inside partition.cfg. I mean  $romfs_partition_enable == 1 can be used inside partition.cfg to include/exclude the romfs

Reply:
I will delete partition_map_romfs.cfg and add Partition Map for ROMFS  in partition_map.cfg using the $romfs_partition_enable variable.